package.json

```diff
@@ -243,6 +243,7 @@
     "playwright-headed": "playwright test --headed",
     "playwright-debug": "playwright test --debug",
     "playwright-report": "playwright show-report",
+    "playwright-e2e-ci": "playwright test --grep @ci",
```

Comment:
similar to the other comment; 'ci' is not a descriptive name.  

we shouldn't name commands based on who uses them (cuz it may change, or there might be multiple users)